products/datagrid/overview.adoc

```diff
@@ -1,4 +1,5 @@
 :awestruct-layout: product-overview
+:awestruct-ignore_export: true
```

Comment:
Any particular reason why we've added the explicit ignore for datagrid overview page here?